Documentation/ceph-filesystem-crd.md

```diff
@@ -187,7 +191,7 @@ See the official cephfs mirror documentation on [how to add a bootstrap peer](ht
 The pools allow all of the settings defined in the Pool CRD spec. For more details, see the [Pool CRD](ceph-pool-crd.md) settings. In the example above, there must be at least three hosts (size 3) and at least eight devices (6 data + 2 coding chunks) in the cluster.
 
 * `metadataPool`: The settings used to create the filesystem metadata pool. Must use replication.
-* `dataPools`: The settings to create the filesystem data pools. If multiple pools are specified, Rook will add the pools to the filesystem. Assigning users or files to a pool is left as an exercise for the reader with the [CephFS documentation](http://docs.ceph.com/docs/master/cephfs/file-layouts/). The data pools can use replication or erasure coding. If erasure coding pools are specified, the cluster must be running with bluestore enabled on the OSDs.
+* `dataPools`: The settings to create the filesystem data pools. Optionally, a pool name can be specified with the `name` field to override the default generated name. If multiple pools are specified, Rook will add the pools to the filesystem. Assigning users or files to a pool is left as an exercise for the reader with the [CephFS documentation](http://docs.ceph.com/docs/master/cephfs/file-layouts/). The data pools can use replication or erasure coding. If erasure coding pools are specified, the cluster must be running with bluestore enabled on the OSDs.
```

Comment:
I would like to be clearer here that while the `name` is optional and that we highly recommend setting the name to prevent issues that can arise from modifying the spec in a way that causes Rook to lose the original pool ordering. I would also suggest adding `name` as a field under `dataPools` here so that we can also document that the actual pool name stored in Ceph for each pool will be `<metadata.name>-<name>`. It will be easier to read the section just on `name` if it is separated. We can still mention here in the `dataPools` section that we recommend setting the `name` parameter on on pools that is documented below.